tests/Microsoft.Bot.Expressions.Tests/ExpressionEngineTests.cs

```diff
@@ -535,7 +535,12 @@ public class ExpressionEngineTests
             Test("max(mul(1, 2), 5) ", 5),
             Test("max(5) ", 5),
             Test("max(4, 5) ", 5),
+            Test("max(createArray(1, 2, 3, 4))", 4),
```

Comment:
Actual the result `4` is a `Double`, what user really want is its original type `Integer`.